tests/io/test_cachehandler_ungridded.py

```diff
@@ -31,6 +33,26 @@ def test_reload_custom(
     assert cache_handler.loaded_data[path.name].shape == aeronetsunv3lev2_subset.shape
 
 
+def test_reload_config(
+    cache_handler: CacheHandlerUngridded, pyaro_testdata: ReadPyaro, tmp_path: Path
+):
+    path = tmp_path / f"{pyaro_testdata.DATA_ID}_concso4.pkl"
```

Comment:
Yes... How else to check if the caching works? I can remove this is you want?

Reply:
It looks like you might be creating it on the fly, which is less of an issue than, for example, adding it to `testdata-minimal`. We've tried to avoid adding pickle files to CI because they can be non-robust with respect to versions of Python and numpy (e.g., #914). I wouldn't not approve this based on doing it, but it's something to be aware of.

Reply:
Then I'll remove this test. I at least knows that the pickling with pyaro works